src/mpid/ch4/netmod/ofi/ofi_capability_sets.h

```diff
@@ -484,7 +484,6 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_get_set_number(const char *set_name)
 #ifdef MPIDI_CH4_OFI_USE_SET_RUNTIME
 #define MPIDI_OFI_SET_NUMBER                MPIDI_OFI_SET_NUMBER_DEFAULT
 #define MPIDI_OFI_ENABLE_RUNTIME_CHECKS     1
-#define MPIDI_OFI_ENABLE_ENDPOINTS_BITS     1
```

Comment:
We do use this internally for scalable endpoints vs. regular endpoints. I'm dragging in @sagarth and/or @shawnccx to say whether or not this would be an issue.

Reply:
This MACRO is not used. Quick search from mpich code today (intel's branch).

Reply:
Yep. It was the `ep_idx` part that was the problem here, not this macro.